frontend/src/components/utils/Footer/index.tsx

```diff
@@ -16,24 +18,42 @@ const useStyles = makeStyles(() => ({
       paddingRight: '1em',
     },
   },
-  link: {
+  contact: {
     color: '#B94630',
     fontWeight: 'bold',
     textAlign: 'center',
+    marginLeft: '4px',
   },
 }));
 const Footer = (): ReactElement => {
-  const { footer, link } = useStyles();
+  const { footer, contact } = useStyles();
   return (
     <footer>
-      <Grid container alignItems="center" justify="center" className={footer}>
+      <Grid container alignItems="center" justify="center" className={footer} direction="column">
         <Grid item>
-          <Typography>
-            Got a question that isn’t answered?{' '}
-            <Link className={link} underline="none" href="mailto:hello@cornelldti.org">
-              Contact us!
-            </Link>
-          </Typography>
+          <Grid container alignItems="center" justify="center">
+            <Grid item>
+              <Typography>Got a question that isn’t answered? </Typography>
+            </Grid>
+            <Grid item>
+              <Typography>
+                <Link className={contact} href="mailto:hello@cornelldti.org">
+                  Contact us!
+                </Link>
+              </Typography>
+            </Grid>
+          </Grid>
+        </Grid>
+        <Grid item>
+          <Link
+            {...{
+              to: `/policies`,
+              style: { color: 'black' },
+              component: RouterLink,
+            }}
+          >
+            <Typography> Privacy Policy </Typography>
```

Comment:
Style Nit: Can this be underlined to distinguish it from the line above (The "Got a question that isn’t answered?" line). I feel like that would make it easier to see.

Reply:
I tried it out, but I think I'll keep it as is because of the hover styling that underlines this link and "contact".